resources/js/components/pages/Settings.vue

```diff
@@ -13,101 +13,16 @@
-                                                token.plainTextToken.substring(
-                                                    token.plainTextToken?.indexOf('|') + 1 ?? 0
-                                                )
-                                            "
-                                        />
-                                    </span>
-                                </div>
-                                <Btn :dusk="`btn__revoke-api${token.name}`" error @click="confirmRevoke(token.name)">
-                                    Revoke
-                                </Btn>
-                            </div>
-                        </div>
-                        <div v-if="walletAccount">
-                            <h1 class="text-xl">Wallet Account</h1>
-                            <p class="mt-1 text-sm text-gray-500 max-w-3xl">
-                                The platform depends on there being a funded wallet daemon account setup and connected
-                                to the platform and blockchain so it can receive transactions from the platform, sign
-                                them on your behalf and then broadcast them to the network. You can get the latest
-                                version of the wallet daemon from this repository:
-                                <a href="https://github.com/enjin/wallet-daemon" target="_blank" class="text-primary">
-                                    https://github.com/enjin/wallet-daemon</a
-                                >
-                                and an overview can be found here:
-                                <a
-                                    href="https://docs.enjin.io/enjin-platform/wallet-daemon"
-                                    target="_blank"
-                                    class="text-primary"
-                                >
-                                    https://docs.enjin.io/enjin-platform/wallet-daemon</a
-                                >
-                            </p>
-                            <div class="flex items-end space-x-4 mt-4">
-                                <FormInput
-                                    v-model="walletAccount"
-                                    class="sm:w-[430px] flex-1 sm:flex-none text-gray-400"
-                                    name="walletAccount"
-                                    disabled
-                                />
-                            </div>
-                        </div>
+                        <SettingsApiTokens />
+                        <SettingsWalletAccount />
                     </div>
                 </div>
 
                 <div class="flex flex-col space-y-4 mt-4">
                     <div class="flex justify-between">
                         <h1 class="text-xl">Settings</h1>
                     </div>
+                    <SettingsResetPassword v-if="appStore.isMultiTenant" />
```

Comment:
I think reset password should be present at all times, right @v16Studios

Reply:
but the mutation is only available on multi tenant package

Reply:
Right, we're using token as login for non multi-tenant right?

Reply:
yes